checks/worker/file_utils.go

```diff
@@ -115,6 +116,58 @@ func CheckFilesContent(shellPathFnPattern string,
 	return nil
 }
 
+// FileContentCbV6 is the callback.
+// The bool returned indicates whether the CheckFilesContent2
+// should continue iterating over files or not.
+type FileContentCbV6 func(path string, content []byte, data FileCbData) (bool, error)
+
+// CheckFilesContentV6 is the same as CheckFilesContent
+// but for use with separated check/policy code.
+func CheckFilesContentV6(shellPathFnPattern string,
```

Comment:
Nit: some of the content in `worker` package seems to better fit inside `fileparser`.

Reply:
right. I agree. But not sure this is in scope for this PR. I think all the `file_utils.go` should live under `fileparser`? @chrismcgehee wdut?